### DISCUSSION TITLE
v1.0.0 tictactoe online game

### DISCUSSION THREAD

**Discussion by AadrianLeo** [AadrianLeo/Tictactoe-Multiplayer-Game#2] [General]
the game is not fully ready and may be unstable.

<hr /><em>This discussion was created from the release <a href='https://github.com/AadrianLeo/Tictactoe-Multiplayer-Game/releases/tag/v1.0.0'>v1.0.0 tictactoe online game</a>.</em>

**Comment by SI-Abid** [comment]
I am willing to contribute.

**Reply by AadrianLeo** [reply]
Sure Please. Currently I am working on the server migration. It may take time as I am little occupied with work. after migration will provide you further updates and code changes.

**Reply by SI-Abid** [reply]
Thank you for letting me know.
